### DISCUSSION TITLE
Is library still actively maintained?

### DISCUSSION THREAD

**Discussion by himrocks33** [srivastavaanurag79/react-native-paper-select#58] [Announcements]
Hi, wondering if this library is still actively maintained?  I opened a PR a couple weeks ago and it has not been reviewed yet.

https://github.com/srivastavaanurag79/react-native-paper-select/pull/56

**Comment by billomore** [comment]
by the description of it looks like a very useful PR

**Comment by srivastavaanurag79** [comment]
Sorry i am not able to work on this project currently.

**Comment by himrocks33** [comment]
@srivastavaanurag79 would you be willing to allow one of the other contributors to merge PRs.  This is a nice project would hate to see it die.

Would one of the active contributors be interested in this?  I would be willing to help with creating PRs where needed for bug fixes and such. 

ahmed (@Bo7mid3)
Thom van den Akker (@Thodor12)
Carlos Loureiro (@CarlosSLoureiro)
b. avianto (@avianto)

**Comment by CarlosSLoureiro** [comment]
Seems a good idea since the maintainer have no time to work on it anymore.

**Comment by srivastavaanurag79** [comment]
Sent **Collaborator** Invitation to: 

Thom van den Akker (@Thodor12)
Carlos Loureiro (@CarlosSLoureiro)

Please provide me with your **npm** username if you accept the invitation.

I use react-native-bob-builder to publish the package

To prepare the package to be published: `npm run prepare`
To release the package: `npm run release`
